src/js/tools/ModulesCollector.ts

```diff
@@ -1,8 +1,12 @@
 import { logger } from '@sentry/utils';
 import { existsSync, mkdirSync, readFileSync, writeFileSync } from 'fs';
 import { posix, sep } from 'path';
+import { env } from 'process';
 
-logger.enable();
+// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+if (env['NODE_ENV'] !== 'test') {
+  logger.enable();
+}
```

Comment:
M: can we actually do that via tests? either mock or something the logger, we should not run code in production that is only used for tests.

Reply:
Okay, yes, that will be better.